packages/client/components/TeamPrompt/TeamPromptTopBar.tsx

```diff
@@ -34,24 +51,27 @@ const TeamPromptTopBar = (props: Props) => {
     graphql`
       fragment TeamPromptTopBar_meeting on TeamPromptMeeting {
         name
+        ...TeamPromptOptionsMenu_meeting
       }
     `,
     meetingRef
   )
 
-  const { name: meetingName } = meeting
+  const {name: meetingName} = meeting
 
   return (
     <MeetingTopBarStyles>
       <HeadingBlock>
         <TeamPromptHeader>
-          <BackButton
-            ariaLabel='Back to Meetings'
-            onClick={() => history.push('/meetings')}
-          />
+          <BackButton ariaLabel='Back to Meetings' onClick={() => history.push('/meetings')} />
           <TeamPromptHeaderTitle>{meetingName}</TeamPromptHeaderTitle>
         </TeamPromptHeader>
       </HeadingBlock>
+      <AvatarGroupBlock>
```

Comment:
Putting this button within the `AvatarGroupBlock` to stay consistent with the styling patterns of the existing `MeetingTopBar`